portfolio/deployVersion.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#Instructions:
+    #From the same directory as this file, run: ./deployVersion.sh
```

Comment:
Prefer to keep the "#" aligned, and have the indentation inside the quote block.